components/posts/posts.php

```diff
@@ -253,6 +274,277 @@ function mm_posts_output_masonry_sizers() {
  * @param  object  $context  The global post object.
  * @param  array   $args     The instance args.
  */
+function mm_post_article( $post, $context, $args ) {
+
+	ob_start(); ?>
+
```

Comment:
What do you think about `mm_posts_before_article` and `mm_posts_after_article` here? These would get passed the same args as `mm_posts_header`, `mm_posts_content`, etc.